### PR TITLE
[chore]: #4231 Update MAINTAINERS (dev branch)

### PR DESCRIPTION
**Pull request by 6r1d**
Let's replace `iroha2-dev` MAINTAINERS file with a fresh version.

### CODE REVIEW

**Review by outoftardis — APPROVED** [review]

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,32 +1,33 @@
 ## Maintainers
 
 Maintainers of Hyperledger Iroha project
```
Comment (WRRicht3r) [inline MAINTAINERS.md:3]:
```suggestion
Maintainers of the Hyperledger Iroha project
```

File: MAINTAINERS.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
-are supposed to help contributors by explain them project details,
+are supposed to help contributors by explaining them project details,
 such as architecture, process, existing issues.
 
 This is the list of maintainers, including their email address for direct communications:
```
Comment (WRRicht3r) [inline MAINTAINERS.md:7]:
```suggestion
This is the list of maintainers, including their email addresses for direct communication:
```

File: MAINTAINERS.md [hunk 3]
```diff
@@ -1,32 +1,33 @@
 ## Maintainers
 
 Maintainers of Hyperledger Iroha project
-are supposed to help contributors by explain them project details,
+are supposed to help contributors by explaining them project details,
 such as architecture, process, existing issues.
```
Comment (WRRicht3r) [inline MAINTAINERS.md:5]:
```suggestion
such as architecture, process, and existing issues.
```

File: MAINTAINERS.md [hunk 4]
```diff
@@ -1,32 +1,33 @@
...
+| Andrey Kostiuchenko   | [@arndey](https://github.com/arndey)                 | kostiuchenko@soramitsu.co.jp    | Developer: Java, Kotlin                                                       |
+| Timur Guskov          | [@gv-timur](https://github.com/gv-timur)             | guskov@soramitsu.co.jp          | Development: Java                                                             |
+| Dmitriy Creed         | [@Cre-eD](https://github.com/Cre-eD)                 | creed@soramitsu.co.jp           | DevSecOps                                                                     |
+| Vasily Zyabkin        | [@BAStos525](https://github.com/BAStos525)           | zyabkin@soramitsu.co.jp         | DevOps                                                                        |
+| Ekaterina Mekhnetsova | [@outoftardis](https://github.com/outoftardis)       | mekhnetsova@soramitsu.co.jp     | Documentation                                                                 |
+| William Richter       | [@WRRicht3r](https://github.com/WRRicht3r)           | ryan.rrr66@gmail.com            | Documentation                                                                 |
```
Comment (WRRicht3r) [inline MAINTAINERS.md:26]:
```suggestion
| William Richter       | [@WRRicht3r](https://github.com/WRRicht3r)           | richter@soramitsu.co.jp            | Documentation                                                                 |
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by mversic — DISMISSED** [review]

**Review by mversic — APPROVED** [review]

File: MAINTAINERS.md [hunk 5]
```diff
@@ -1,32 +1,33 @@
...
+| --------------------- | ---------------------------------------------------- | ------------------------------- | ----------------------------------------------------------------------------- |
+| Makoto Takemiya       | [@takemiyamakoto](https://github.com/takemiyamakoto) | takemiya@soramitsu.co.jp        | Product vision                                                                |
+| Dmitri Venger         | [@dmitrivenger](https://github.com/dmitrivenger)     | venger@soramitsu.co.jp          | Project manager                                                               |
+| Bogdan Mingela        | [@Mingela](https://github.com/Mingela)               | mingela@soramitsu.co.jp         | Iroha Team Lead                                                               |
+| Aleksandr Petrosyan   | [@appetrosyan](https://github.com/appetrosyan)       | ap886@cantab.ac.uk              | Iroha 2 architect                                                             |
+| Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust                                                             |
```
Comment (Arjentix) [inline MAINTAINERS.md:15]:
```suggestion
| Daniil Polyakov       | [@Arjentix](https://github.com/Arjentix)             | daniil.polyakov@soramitsu.co.jp | Development: Rust, C++                                                             |
```

As I can see we're adding additional languages here

**Review by mversic — APPROVED** [review]

**Review by 0x009922 — CHANGES_REQUESTED** [review]
I guess @s8sato should be mentioned too?

**Comment by 6r1d** [comment]
Yes, most definitely.

**Review by mversic — APPROVED** [review]

**Review by outoftardis — APPROVED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Review by 0x009922 — APPROVED** [review]

**Pull request merged**